### ISSUE TITLE
new user has to log in twice

### ISSUE THREAD

**Issue by smaddock** [splittingred/modActiveDirectory#6]
When logging in with an AD account that has not been used on a given MODx install, the first time that user logs in they just get a white screen. If they reload the page and try to log in again everything works correctly.

**Comment by BigBlockStudios** [comment]
Did you figure this one out? I'm seeing the same problem here.

**Comment by smaddock** [comment]
No, although I haven't tried again recently incase it was fixed in an update.

**Comment by BigBlockStudios** [comment]
No - it appears to have not been updated in about 3 years.  :(

**Comment by BigBlockStudios** [comment]
I found it: core/components/activedirectory/elements/events/onusernotfound.php
comment out: line 57,

```
    //$user->Profile =& $profile;
```
